src/babel.config.js

```diff
@@ -1,6 +1,5 @@
 /* @flow */
 
-// eslint-disable-next-line import/no-commonjs
-module.exports = {
-    extends: 'grumbler-scripts/config/.babelrc-browser'
+ module.exports = {
+    "extends": "@krakenjs/babel-config-grumbler/babelrc-node"
```

Comment:
Should this be `"@krakenjs/babel-config-grumbler/babelrc-browser"`?

Reply:
Good call! It should!

Reply:
updated